### ISSUE TITLE
Add showSearch to dropdown properties grid

### ISSUE THREAD

**Issue by meysammahfouzi** [rainlab/builder-plugin#144]
It seems that it's not possible to set `showSearch` from the properties grid of a [dropdown](https://octobercms.com/docs/backend/forms#field-dropdown):

![dropdown](https://cloud.githubusercontent.com/assets/14848008/23625373/29da0138-02be-11e7-8dcc-c39a27b1b892.PNG)

Could you please add it? Otherwise we have to manually update the yaml file:

```
status:
    label: Blog Post Status
    type: dropdown
    showSearch: false
```

**Comment by LukeTowers** [comment]
Feel free to make a PR including this. See https://github.com/rainlab/builder-plugin/blob/master/classes/StandardControlsRegistry.php#L127

**Comment by bennothommo** [comment]
This is now available as of ef437a00d271bdd1dd9ca4e929c3a91589124d75.
